README.md

```
@@ -45,13 +47,45 @@
-pnpm frontend:dev
+# Run both frontend and backend (concurrently)
+pnpm dev
 
-# Run the backend development server
+# Or run them separately
+pnpm frontend:dev
 pnpm backend:dev
 ```
 
+Ports:
+- Frontend: `http://localhost:3000`
```

markdownlint-cli2 (0.17.2)

Lists should be surrounded by blank lines

(MD032, blanks-around-lists)